udata/core/dataset/csv.py

```diff
@@ -74,6 +78,8 @@ class ResourcesCsvAdapter(csv.NestedAdapter):
         'created_at',
         'modified',
         ('downloads', lambda o: int(o.metrics.get('views', 0))),
+        ('harvest.created_at', lambda o: o.harvest and o.harvest.created_at),
```

Comment:
It would be nice to be able to say that a resource has been harvested or not, but I don't think we have a dedicated info at the resource level.

Reply:
Yep, but no we don't.

I tried to expose a `dataset.is_harvested` field based on `dataset.harvest.remote_id` presence. It works but it feels flaky. Decided not too :-)

Maybe we could add a dedicated property on `Dataset` with a well defined and contained logic, then expose it to the dataset and/or resources catalog. Feels like another PR though.